templates/tools/dockerfile/interoptest/grpc_interop_aspnetcore/build_interop.sh.template

```diff
@@ -25,14 +25,11 @@
   cp -r /var/local/jenkins/service_account $HOME || true
 
   cd /var/local/git/grpc-dotnet
-  
+
   # If needed, update dotnet SDK and put it on path
   ./build/get-dotnet.sh
-  if [ -f $HOME/.dotnet/dotnet ]
-  then
-    ln -s $HOME/.dotnet/dotnet /usr/local/bin/dotnet
-  fi
-  
+  source ./activate.sh
```

Comment:
There is a circular dependency here between this PR and https://github.com/grpc/grpc-dotnet/pull/205. activate.sh doesn't exist in the grpc-dotnet repo yet.

What is the best way to resolve this? Would merging activate.sh in a separate PR help?

Reply:
Activate.sh would only make sense if we made the change to install to a local .dotnet/ directory. Is it possible to merge while ignoring failing tests? I'm thinking we might need to merge the changes in grpc-dotnet without the interop tests passing and then merge this as a reaction PR.

Reply:
We can't. An admin (@jtattermusch?) should be able to.